docs/feature-guides/analyze-with-check-checker.md

```diff
@@ -3,7 +3,14 @@ title: Analyze with Check-Checker
 ---
 
 The check-checker is a static analysis pass you can use to help find potential vulnerabilities in your contracts.
-To enable this pass, add the following lines to your Clarinet.toml file:
+
+*Topics*:
+
+- [x] Enable static analysis pass
```

Comment:
These `[x]` would open up the checked mark, but what is the intent? A more impactful alternative is making each line-item hyperlink to individual sections on the page so that readers can jump to a specific section instead of scrolling.

Reply:
The initial idea was to add hyperlinks, but Docusaurus defaults those to the right nav TOCs. I could only add topics to the headings (H1s and H2s). 
So, I removed the hyperlinks but added a standalone topic to understand what is covered in a specific article.

Reply:
Making a text with hyperlinks tricking the left nav seems odd (perhaps I am interpreting it incorrectly); there are many places with hyperlinks in the body of the page, though, and that seems all kosher. Do we have a preview of this, so I can see it in action?

Reply:
Before saving the file:

<img width="548" alt="image" src="https://user-images.githubusercontent.com/55154433/220985254-2f1cedc2-d76c-4150-90cb-3e185e87a3c4.png">

After saving the file:

<img width="648" alt="image" src="https://user-images.githubusercontent.com/55154433/220985433-b2348ed3-9727-4055-981e-de30f5732afe.png">

Reply:
This issue is now resolved by turning off auto save for TOC in VS code. Thanks @hugocaillard.